src/com/xilinx/rapidwright/rwroute/PartialRouter.java

```diff
@@ -78,6 +85,46 @@ protected void addStaticNetRoutingTargets(Net staticNet){
 		}
 	}
 	
+	private List<SitePinInst> findUnroutedSinks(Net net, List<SitePinInst> sinks) {
+	    Set<Node> usedNodes = new HashSet<>();
+	    List<SitePinInst> unroutedSinks = new ArrayList<>();
+	    for(PIP p : net.getPIPs()) {
+	        usedNodes.add(p.getStartNode());
+	        usedNodes.add(p.getEndNode());
+	    }
+	    for(SitePinInst sink : sinks) {
+	        if(!usedNodes.contains(sink.getConnectedNode())) {
+	            unroutedSinks.add(sink);
+	        }else {
+	            sink.setRouted(true);
+	        }
+	    }
+	    return unroutedSinks;
+	}
```

Comment:
Would a more comprehensive approach be to tell `PartialRouter` what pins you want it to route?